arrowFunctions.js

```diff
@@ -13,9 +15,32 @@ function modifyArray(nums) {
 function modifyArrayX(nums) {
   for (let i = 0; i < nums.length; i++) {
     if (nums[i] % 2 === 0)
-      nums[i] * 2;
+     nums[i] *= 2; 
     else
-      nums[i] * 3;
+     nums[i] *= 3;
   }
-  return nums;
-}
\ No newline at end of file
+   return nums;
+}
+ 
+
+// ES6 for of
+
+function modifyArray6(nums) {
+for (const n of nums) {
+  if (n % 2 != 0) {
+   var cur = n * 3
+  } else {
+    var cur = n * 2;
+  }
+  var curArr = console.log(cur);
+
+} 
+
+}
+
+
+var p = modifyArray6(nums);
+console.log(p);
+
+
+
```

Comment:
@Ayopal This code snippet doesn't return an array and again does not use arrow functions.

Reply:
code removed and indentation adjusted;

Please kindly advice on a good snippet/plug-in to use for auto-indent